crates/nu-command/src/filesystem/umkdir.rs

```diff
@@ -0,0 +1,76 @@
+use nu_engine::env::current_dir;
+use nu_engine::CallExt;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, IntoInterruptiblePipelineData, PipelineData, ShellError, Signature,
+    SyntaxShape, Type, Value,
+};
+
+use uu_mkdir::mkdir;
+
+#[derive(Clone)]
+pub struct UMkdir;
+
+const IS_RECURSIVE: bool = true;
+// This is the same default as Rust's std uses:
+// https://doc.rust-lang.org/nightly/std/os/unix/fs/trait.DirBuilderExt.html#tymethod.mode
+const DEFAULT_MODE: u32 = 0o777;
+
+impl Command for UMkdir {
+    fn name(&self) -> &str {
+        "umkdir"
+    }
+
+    fn usage(&self) -> &str {
+        "Create directories, with intermediary directories if required."
```

Comment:
```suggestion
        "Create directories, with intermediary directories if required using uutils/coreutils mkdir."
```
we've been trying to be consistent with adding this part to the coreutils commands